src/components/UI/Tab/Tab.js

```diff
@@ -3,15 +3,15 @@ import React from 'react';
 
 import styles from './Tab.module.scss';
 
-export const Tab = ({label, onClick}) => {
+export const Tab = ({onClick, label, style}) => {
```

Comment:
Remove the `style` prop and handle it in the scsss module.

Reply:
done